src/main/java/com/example/pokebattlez/battle/controller/Battle.java

```diff
@@ -17,56 +20,66 @@
     private final List<PlayerAction> playerActions = new ArrayList<>();
 
     private BattleService battleService;
-
-    public void setBattleService(BattleService battleService) {
-        this.battleService = battleService;
-    }
+    private static final BattleUtils battleUtils = new BattleUtils(); // Ask about this
 
     public void registerPokemonTeam(Long trainerId, List<Pokemon> pokemon) {
```

Comment:
Maybe pokemon**s** to indicate that it's a collection of them?

Reply:
I get what you mean, but grammatically "pokemon" is both singular and plural.....